### PR TITLE
Grafana not installing when not running on jessie

### LINKED ISSUE #239: Grafana on Ubuntu 17.10 fails to install
Hi,

I have set up a Minimal Ubuntu Server 17.10 system in a virtual machine on ESXi 6.5 (just in case it's relevant, or anyone wants to replicate my setup!)

When attempting to install InfluxDB & Grafana, it fails.

By Looking at the script, it appears to be bypassing the steps regarding the repository and keyserver for Grafana, because these are wrapped within first a check that it's `jessie` and secondly that it's on a Pi. As I'm failing both of these checks, apt is never informed of where to get Grafana from, so fails on `apt -y install grafana`

InfluxDB appears to be installing and running OK.

### PR DESCRIPTION
**Pull request by ConfusedTA**
When base operating system is not Jessie, Grafana repository was not being added to apt

Fixes #239

Signed-off-by: Garry Mitchell <garrymitchell@my105e.com> (github: ConfusedTA)

### CODE REVIEW

**Comment by ConfusedTA** [comment]
I was getting permission denied errors when trying to use the same syntax as 1188 - however, I've changed it and submitted an updated PR (I hope - I'm fairly new to github!)

File: openhabian-setup.sh [hunk 1]
```diff
@@ -1194,11 +1194,13 @@ influxdb_grafana_setup() {
   if [ $FAILED -eq 1 ]; then echo -n "FAILED "; else echo -n "OK "; fi
   cond_echo ""
 
-  echo -n "Grafana (fg2it)... "
-  if is_jessie; then
-    if is_pione || is_pizero || is_pizerow; then GRAFANA_REPO_PI1="-rpi-1b"; fi
```
Comment (ThomDietrich) [inline openhabian-setup.sh]: This line has to stay!

File: openhabian-setup.sh [hunk 2]
```diff
@@ -1194,11 +1194,13 @@ influxdb_grafana_setup() {
...
+  echo -n "Grafana... "
+  if is_pi; then
     echo "deb https://dl.bintray.com/fg2it/deb${GRAFANA_REPO_PI1} jessie main" > /etc/apt/sources.list.d/grafana-fg2it.list || FAILED=2
     cond_redirect apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 379CE192D401AB61 || FAILED=2
+  else
+    cond_redirect wget -O - https://packagecloud.io/gpg.key | sudo apt-key add - || FAILED=2
```
Comment (ThomDietrich) [inline openhabian-setup.sh]: please remove `sudo` here

File: openhabian-setup.sh [hunk 3]
```diff
@@ -1194,11 +1194,13 @@ influxdb_grafana_setup() {
   if [ $FAILED -eq 1 ]; then echo -n "FAILED "; else echo -n "OK "; fi
   cond_echo ""
 
-  echo -n "Grafana (fg2it)... "
-  if is_jessie; then
-    if is_pione || is_pizero || is_pizerow; then GRAFANA_REPO_PI1="-rpi-1b"; fi
+  echo -n "Grafana... "
```
Comment (ThomDietrich) [inline openhabian-setup.sh]: Please add after this line, before the if:

```shell
rm -f /etc/apt/sources.list.d/grafana*
```

File: openhabian-setup.sh (hunk 1 above)
Comment (ConfusedTA) [reply openhabian-setup.sh]: Yes, looks like it! Will add it back!

File: openhabian-setup.sh (hunk 2 above)
Comment (ConfusedTA) [reply openhabian-setup.sh]: Will do!

**Comment by ThomDietrich** [comment]
Yes you have done everything right. After those two comments we are ready to merge.
Please sign-off your commits just as you did in your PR description ;)

**Comment by ConfusedTA** [comment]
Done, and remembered to add the sign-off!

**Comment by ThomDietrich** [comment]
Amazing. Here we go.
Thanks! 🎉

**Pull request merged**